client/state/login/actions/login-user-with-security-key.js

```diff
@@ -59,7 +60,34 @@ export const loginUserWithSecurityKey = () => ( dispatch, getState ) => {
 				dispatch( updateNonce( twoFactorAuthType, twoStepNonce ) );
 			}
 
-			const error = getErrorFromHTTPError( httpError );
+			const errorMessages = {
```

Comment:
In other circumstances, I'd recommend declaring this object outside this Redux thunk. However, not this time - we're using the non-reactive version of `translate()`, which won't trigger updates automatically if the locale changes or new translations are loaded. So, it's better to run fresh `translate()` calls every time this happens.

So, nothing to do here, just wanted to point it out if it's not obvious.